samples/notebooks/python/1-basic-loading-the-kernel.ipynb

```diff
@@ -24,8 +24,17 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from init import *\n",
-    "import semantic_kernel as sk"
+    "!python3 -m pip install ../../../python"
```

Comment:
might want to call out that if on Windows, this will most likely be 

`!python -m pip install ../../../python`

Reply:
I switched them to just `python` and they still run for me (on WSL), so let's go with that for now

Reply:
I think this is fine. This is the default for most python things I come across and most python users on Unix know to switch to python3 syntax when things don't work right off the bat.